server/src/helpers/errors.ts

```diff
@@ -1,6 +1,6 @@
 import { Error } from 'rosetta-client';
 
-interface E {
+export interface RosettaError {
```

Comment:
More clear naming 👍🏽

Reply:
I actually don't think this needs to be exported though? Initially I was trying to use it in `Controller.ts`, but after making the `isRosettaError` function I think this can be unexported / private again.

Reply:
Yes, if it's only used in this fil, there is no need to export it.

Reply:
Fixed here: https://github.com/gear-foundation/integrations-rosetta-api/pull/57/commits/dc57af33d27cc26ec9154b24711e618a8b276f15